### DISCUSSION TITLE
How to disable animation for echasnovski/mini.indentscope plugin?

### DISCUSSION THREAD

**Discussion by ljudina** [LazyVim/LazyVim#1520] [Q&A] [answered]
Hi, 
I'm having difficulties disabling animation for matching braces for mini.indentscope plugin in config folder
`lua\config\indentscope.lua`
```
return {
  "echasnovski/mini.indentscope",
  opts = {
    draw = {
      animation = function()
        return 0
      end,
    },
  },
}
```
But the animation is still running.
Can you help me out with this?
Thanks,
Marko

**Comment by jyuan0** [comment, marked as answer]
I think you should be putting the file in `lua\plugins\`, rather than `lua\config\`.

And this code to disable the animation works for me:
```lua
return {
  "echasnovski/mini.indentscope",
  opts = {
    draw = {
      animation = require("mini.indentscope").gen_animation.none(),
    },
  },
}
```

**Comment by ljudina** [comment]
Hi jyuan0,

Thanks for sharing solution. It worked when I have moved file into `lua\plugins\` folder.
